server/game/cards/04.3-NN/OldManMcDroste.js

```diff
@@ -0,0 +1,44 @@
+const DudeCard = require('../../dudecard.js');
+const PhaseNames = require('../../Constants/PhaseNames.js');
+
+class OldManMcDroste extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'any',
+            condition: () => this.isOutOfTown() && this.isAtDeed(),

```

Comment:
To check if dude is in out of town deed, use this:
```suggestion
            condition: () => this.isAtDeed('out-town'),

```